src/main/kotlin/gg/flyte/christmas/util/StringUtil.kt

```diff
@@ -0,0 +1,29 @@
+package gg.flyte.christmas.util
+
+fun String.toSmallText(): String = lowercase()
```

Comment:
As I do agree that it doesn't always look good, another idea might be, since you already have a resource pack, using a custom font which adds small numbers. I think that's the best idea since otherwise there is a loss in consistency in some messages.

Reply:
> As I do agree that it doesn't always look good, another idea might be, since you already have a resource pack, using a custom font which adds small numbers. I think that's the best idea since otherwise there is a loss in consistency in some messages.

I will see if this can be looked into. That also would mean reverting 99% of the small-text changes you have made here, but I don't see that being too difficult. For now though I would remove it. 

Fortunately, since the client already pixelates any character (small or regular), it isn't too inconsistent when a random player name is thrown into the text.

Reply:
> I will see if this can be looked into. That also would mean reverting 99% of the small-text changes you have made here, but I don't see that being too difficult. For now though I would remove it.

Shouldn't be too difficult yeah.

> Fortunately, since the client already pixelates any character (small or regular), it isn't too inconsistent when a random player name is thrown into the text.

It's still quite noticeable and inconsistent, so a custom font might be a nice idea.